eogrow/pipelines/merge_samples.py

```diff
@@ -29,7 +29,12 @@ class Schema(Pipeline.Schema):
             description="Dictionary of all features for which samples are to be merged."
         )
         include_timestamp: bool = Field(False, description="Whether to also prepare an array of merged timestamps.")
-        id_filename: Optional[str] = Field(description="Filename of array holding patch id of concatenated features")
+        id_filename: Optional[str] = Field(
+            description=(
+                "Filename of array holding patch ID of concatenated features. The patch ID is the index of the patch in"
+                " the final patch list, any filtration of the patch list will impact the results."
```

Comment:
Is it now any harder to obtain the eopatch of interest if the index of the patch ID is affected by the filtration?

Reply:
yes, but to be fair it wasn't trivial before either :)

Reply:
Before, I had to look at the index I was using, and find the eopatch name corresponding to that index in the grid file. How would one get the eopatch now? e.g. if I have an eopatch name and I want the index for that name, but there is some filtration going on.

Reply:
it should correspond to the index of the name in the `finished.json` I believe